models/prices/ethereum/prices_ethereum_tokens.sql

```diff
@@ -1642,3 +1644,5 @@ where contract_address not in (
     ,0x22acaee85ddb83a3a33b7f0928a0e2c3bfdb6a4f   -- PRXY
     ,0xe94b97b6b43639e238c851a7e693f50033efd75c   -- RNBW 
 )
+
+I have read the CLA Document and I hereby sign the CLA
```

Comment:
@0xpibbles sorry for the confusion. this should not be in the code itself. please do remove here

you need to copy/paste:
I have read the CLA Document and I hereby sign the CLA

directly into a gh comment on the PR, at the bottom of the conversation page

Reply:
ahh i see, thanks for your patience and helping me out!